### PR TITLE
feat(github): Add Releases and Release Assets

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: plugins/source/github/docs/tables/github_releases.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+|tag_name (PK)|String|
+|target_commitish (PK)|String|
+|name|String|
+|body|String|
+|draft|Bool|
+|prerelease|Bool|
+|make_latest|String|
+|discussion_category_name|String|
+|generate_release_notes|Bool|
+|id|Int|
```
Comment (hermanschaaf) [inline plugins/source/github/docs/tables/github_releases.md:30]: Would id maybe make a better PK than `tag_name` and `target_commitish`? I haven't used the API so don't really know, just asking the question

File: plugins/source/github/codegen/recipes/repositories.go [hunk 2]
```diff
@@ -30,9 +30,30 @@ func Repositories() []*Resource {
...
+	return []*Resource{
+		repo, alert, sec,
+		{
+			Service:      "repositories",
+			TableName:    "releases",
+			Multiplex:    orgMultiplex,
+			SubService:   "releases",
+			Struct:       new(github.RepositoryRelease),
+			PKColumns:    []string{"tag_name", "target_commitish"},
+			ExtraColumns: codegen.ColumnDefinitions{orgColumn},
```
Comment (candiduslynx) [inline plugins/source/github/codegen/recipes/repositories.go:44]:
```suggestion
			ExtraColumns: codegen.ColumnDefinitions{orgColumn, repositoryID},
```

File: plugins/source/github/codegen/recipes/repositories.go [hunk 3]
```diff
@@ -30,9 +30,30 @@ func Repositories() []*Resource {
...
+			Relations:    []string{"Assets()"},
+		},
+		{
+			Service:      "repositories",
+			TableName:    "release_assets",
+			Multiplex:    orgMultiplex,
+			SubService:   "assets",
+			Struct:       new(github.ReleaseAsset),
+			PKColumns:    []string{"id"},
+			ExtraColumns: codegen.ColumnDefinitions{orgColumn},
```
Comment (candiduslynx) [inline plugins/source/github/codegen/recipes/repositories.go:54]:
```suggestion
			ExtraColumns: codegen.ColumnDefinitions{orgColumn, repositoryID},
```

File: plugins/source/github/docs/tables/github_releases.md (hunk 1 above)
Comment (candiduslynx) [reply plugins/source/github/docs/tables/github_releases.md:30]: +1

**Review by candiduslynx — CHANGES_REQUESTED** [review]
need to add repositoryID col 9parent ID col, basically), see suggections

File: plugins/source/github/codegen/recipes/repositories.go (hunk 3 above)
Comment (bbernays) [reply plugins/source/github/codegen/recipes/repositories.go]: Can't do this... Assets is the grandchild of Repository

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**